### PR TITLE
Task one

### PR DESCRIPTION
**Pull request by Abrams8**

### CODE REVIEW

File: src/main/java/com/abramchik/taskOnePatterns/behavioral/chainOfResponsibility/Notifier.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package com.abramchik.taskOnePatterns.behavioral.chainOfResponsibility;
+
+public abstract class Notifier {
+    private int priority;
+    private Notifier nextNotifier;
+
```
Comment (ALnOUA) [inline src/main/java/com/abramchik/taskOnePatterns/behavioral/chainOfResponsibility/Notifier.java]: Also, we can use @fieldsdefaults for decreasing repeatable code

File: src/main/java/com/abramchik/taskOnePatterns/behavioral/chainOfResponsibility/Notifier.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    public Notifier(int priority) {
+        this.priority = priority;
+    }
+
+    public void setNextNotifier(Notifier nextNotifier) {
+        this.nextNotifier = nextNotifier;
+    }
```
Comment (ALnOUA) [inline src/main/java/com/abramchik/taskOnePatterns/behavioral/chainOfResponsibility/Notifier.java]: Use lombok for setters, getters, constructors etc

**Review by ALnOUA — CHANGES_REQUESTED** [review]

File: src/main/java/com/abramchik/taskOnePatterns/behavioral/chainOfResponsibility/Notifier.java (hunk 1 above)
Comment (Abrams8) [reply src/main/java/com/abramchik/taskOnePatterns/behavioral/chainOfResponsibility/Notifier.java]: done

File: src/main/java/com/abramchik/taskOnePatterns/behavioral/chainOfResponsibility/Notifier.java (hunk 2 above)
Comment (Abrams8) [reply src/main/java/com/abramchik/taskOnePatterns/behavioral/chainOfResponsibility/Notifier.java]: done

**Review by ALnOUA — APPROVED** [review]

**Pull request merged**